R/bbr.R

```diff
@@ -321,9 +321,41 @@ bbi_help <- function(.cmd_args=NULL) {
 #'   model. `submit_model()` and `submit_models()` also support passing a
 #'   configuration file via the `.config_path` argument.
 #'
+#'  **Notes on `.nonmem_dir`**
+#'
+#'   `bbi` tries to detect subdirectories for different NONMEM versions, along
+#'   with a license file and executable within the subdirectory.
+#'
+#'   `.nonmem_dir` should point to the **parent** directory that contains one
+#'   or more NONMEM installations.  It should *not* point to the installation
+#'   directory itself.  Here is an example:
+#'
+#' ```
+#'     /opt/NONMEM/
+#'     |-- nm74
+#'     |   |-- license
+#'     |   |-- run
+#'     |   |-- source
+#'     |   | ...
+#'     |   `-- util
+#'     `-- nm75
+#'         |-- license
+#'         |-- run
+#'         |-- source
+#'         | ...
+#'         `-- util
+#' ```
+#'
+#'   `/opt/NONMEM/` contains two NONMEM installations, one for NONMEM 7.5
+#'   and one for NONMEM 7.4.  In this case, you should pass `"/opt/NONMEM"`
+#'   as `.nonmem_dir` and either `"nm74"` or `"nm75"` as `.nonmem_version`.
+#'
+#'
 #' @param .dir Path to directory to run `init` in (and put the resulting
 #'   `bbi.yaml` file)
-#' @param .nonmem_dir Path to directory with the NONMEM installation.
+#' @param .nonmem_dir Path to directory with the NONMEM installation. This
+#'   should be the **parent** directory that contains that installation. See
```

Comment:
[ This was suppose to be a part of my first review, but I don't see it; I must have forgot to post it.  Sorry. ]

The second sentence would point people in the right direction, but I find the combination of the first and second sentences a bit confusing.  To me it almost reads as the though the second sentence corrects what the first just said.  (That all depends on how the reader interprets "directory with", which I suppose is probably the core of the original issue.)

I'd be in favor of reducing the first two sentences down to one.  Perhaps "Path to parent directory containing one or more NONMEM installations. See [...]".